### ISSUE TITLE
Redesign the website (and the Git repository)

### ISSUE THREAD

**Issue by LeCodingWolfie** [Fultonium-Entertainment/website#5]
## Issue

The website's (amateur-ish) design was developed from scratch solely through code, without any care in the world, and no knowledge or experience on UI/UX or design in general. [^1]

Initially a portfolio, now has dramatically changed into an aspiring streaming platform, the design has remained the same, which makes the UX look odd, and the code messy. In fact, every change that has been requested for the website, has made the code messier, and the Git repository *bloated*, occupying great amounts of disk space from the back-and-forth of media files, and due to the number of branches, stashes, and commits that have been created locally to keep up with the mess, which is a really messy experience that also impairs commits to be uploaded publicly.

In a nutshell, programming the website has been a dirty experience due to the instability of changes requested every now and then, which I hope to stop by both redesigning the website and cleaning up the repository and the code, a win-win situation.

____

Currently, @joaoiacillo's Figma design can be implemented into code while improving it on the process with the original look of the website; whereas the Git repository can be archived towards a *less* bloated, cleaner repository, w/o (some of) the (bigger) media files, which can be taken to somewhere else.

<sup>The serverworm says hi btw :worm:</sup>

[^1]: Similar to my [`aesthetic`][a] project, which turned out better.

[a]: https://github.com/LeCodingWolfie/aesthetic
